src/App.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
```

Comment:
Please remove react's default files and configuration to personalize your instance.